docs/authentication.md

```diff
@@ -35,15 +35,15 @@ Examples of acceptable exceptions include endpoints like `/user` that provide in
 
 To validate a token, check that:
 
-- The signature can be verified with the public key of our Auth0 tenant
-- The `nbf` claim is in the past
-- The `exp` claim is **not** in the past
-- The `aud` claim is equal to `https://api.publiq.be`
-- The `iss` claim is equal to either (depending on what environment your API is running on):
-  - `https://publiq-acc.eu.auth0.com/`
-  - `https://publiq-test.eu.auth0.com/`
-  - `https://publiq.eu.auth0.com/` 
-- The `https://publiq.be/publiq-apis` claim is present and contains a value representing your api
+-   The signature can be verified with the public key of our Auth0 tenant
+-   The `nbf` claim is in the past
+-   The `exp` claim is **not** in the past
+-   The `aud` claim is equal to `https://api.publiq.be`
+-   The `iss` claim is equal to either (depending on what environment your API is running on):
+    -   `https://publiq-acc.eu.auth0.com/`
```

Comment:
this is not a new issue, but this is not 100% correct: if we document that the client should get tokens from account[acc|test].uitid.be, the issuer will be https://acccount[acc|test].uitid.be

Reply:
Right! I tested it with a token from udb to get the values but that uses the old/wrong URL indeed 😂 I'll change it

Reply:
See 03dcbec